curl.mk

```diff
@@ -15,7 +15,7 @@ ifneq ($(wildcard $(BUILD_WORK)/curl/.build_complete),)
 curl:
 	@echo "Using previously built curl."
 else
-curl: curl-setup openssl libssh2 nghttp2 libidn2
+curl: curl-setup openssl libssh2 nghttp2 libidn2 libunistring
```

Comment:
When adding new dependencies to a package, please make sure to add the dependency to its respective control file.

Reply:
forgot about those again kek